### DISCUSSION TITLE
Add ability for a policy to say if it is restricted to some regions

### DISCUSSION THREAD

**Discussion by howbazaar** [cloud-custodian/cloud-custodian#7436] [General]
### Describe the feature

Using c7n as a library, I'd like to be able to ask a Policy object for the regions it is restricted to running against.

There are at least two ways to express this:
 * as a top level region field on the policy
 * as a condition of the policy

Being able to have a method on the Policy itself would keep the code in c7n.

**Comment by kapilt** [comment]
c7n doesn't support library usage, its intended for use an an application. the region attribute is a legacy attribute, policy conditions are the modern mechanism to support region and other conditions (time, account, etc).
